zanata-war/src/main/webapp/admin/roledetail.xhtml

```diff
@@ -13,13 +13,6 @@
       .groups label {
         padding-left: 0.40625em;
       }
-
-      .groups input[type=checkbox] {
-        width: 1em;
-        height: 1em;
-        vertical-align: middle;
-        margin-bottom: 0.20625em;
-      }
```

Comment:
- [ ] @aeng Are these styles in assets now? I notice there are several places that have the same set of styles.